managed/src/main/java/com/yugabyte/yw/controllers/CustomerConfigController.java

```diff
@@ -39,6 +39,13 @@ public Result create(UUID customerUUID) {
     if (errorJson.size() > 0) {
       return ApiResponse.error(BAD_REQUEST, errorJson);
     }
+    if (formData.get("name").asText().equals("S3") &&
+      formData.get("data").get("AWS_ACCESS_KEY_ID") != null) {
+      errorJson = configValidator.validateS3DataContent(formData, customerUUID);
+    }
```

Comment:
Nit: Change this and all the above validate errors to append to errorJsn or errorjson[]
Will the frontend handle error Json  with multiple errors?
If it can, that is a better validation experience. This is because then user can see and fix all errors together not one by one. 
If frontend cannot be changed easily to do this then leave it.

Reply:
I checked with FE folks, as of now there is no handling of errors in the UI. Everything is logged into the browser console. If that is the case then browser can handle it pretty well.

Reply:
Its not checking that even if you give any name it would be take that. We made sure that bucket name should start with "s3://", hence made changes accordingly.

Reply:
I am looking for something like:
```
    validators.add(new PreflightS3BucketValidator(STORAGE_TYPE, NAME_S3, BACKUP_LOCATION_FIELDNAME, ...));
```
and the logic for preflight checks should be in there. 
lets talk if you think it is not possible.